README.md

```diff
@@ -2,82 +2,18 @@
 
 ### Overview
 
-A Data Explorer UI allows for faceted search. For example,
-[Boardwalk](https://commons.ucsc-cgp-dev.org/boardwalk) has facets Analysis
-Type, Center Name, etc.
+This repo contains indexers which index a dataset into Elasticsearch, for
+use by the [Data Explorer UI](https://github.com/DataBiosphere/data-explorer).
 
-A dataset may have hundreds of fields. The dataset owner
-must [curate a list](https://github.com/DataBiosphere/data-explorer/blob/master/dataset_config/1000_genomes/ui.json)
-of the most important fields (age, gender, etc).
+For each dataset, two Elasticsearch indices are created:
 
-A dataset has a notion of `participant_id` and (optional) `sample_id`. The
-columns in BigQuery tables which represent these IDs are configured in [`bigquery.json`](https://github.com/DataBiosphere/data-explorer-indexers/blob/master/dataset_config/template/bigquery.json#L17-L18).
+1. DATASET_NAME: Each Elasticsearch document represents a participant. The document id is participant id.  
+A participant can have
+zero or more samples. Within a participant document, a sample is a
+[nested object](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html#_using_literal_nested_literal_fields_for_arrays_of_objects). Each nested object has a `sample_id` field.
+1. DATASET_NAME_fields: Each document represents a field. The document id is name of the Elasticsearch field from the first index. Example fields are age, gender, etc. This index powers field search. TODO: Include screenshot.
 
 
-### Participant Indexing
-`participant_id` is used as the primary key in the index and ties information
-together from different sources. Say there are facets for age and weight; age
-and weight are stored in separate BigQuery tables, all with a `participant_id`
-column. First, age table is indexed. An Elasticsearch document is created for
-each `participant_id` with document id = `participant_id`. A document would look
-like:
-```
-{
-  "myproject.mydataset.table1.age": "30",
-}
-```
-
-Then, the weight table is indexed. The Elasticsearch documents will get a new
-weight field:
-```
-{
-  "myproject.mydataset.table1.age": "30",
-  "myproject.mydataset.table2.weight": "140",
-}
-```
-
-`participant_id` will be used to figure out which document to update.
-
-### Sample Indexing
-`participant_id` is also used as the foreign key for samples, which are stored
```

Comment:
I'd prefer to leave some version of this in, otherwise I don't think it's very clear how sample indexing works. The description of `samples` as nested objects and that the dataset curator has to specify participant and sample ID columns are fine, but I think we need to more fully explains how fields are split up between participants and samples.

Reply:
Done, thanks. PTAL